src/thing_physics.c

```diff
@@ -876,6 +876,40 @@ unsigned short get_slide_z_coord(const struct Thing *thing, const struct Coord3d
   }
   return ((((z_pos + clipbox_size) & 0xFFFFFF00) - clipbox_size) - 1);
 }
+
+TbBool move_object_to_nearest_free_position(struct Thing *thing)
```

Comment:
Did you intend to have this function in this PR too?

Reply:
yes, the other function calls this one